test/powershell/Modules/Microsoft.PowerShell.Core/Exec.Tests.ps1

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'Switch-Process tests for Unix' {
+    BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if (-not [ExperimentalFeature]::IsEnabled('PSExec') -or $IsWindows)
+        {
+            $PSDefaultParameterValues['It:Skip'] = $true
+            return
+        }
+    }
+
+    AfterAll {
+        $global:PSDefaultParameterValues = $originalDefaultParameterValues
+    }
+
+    It 'Exec alias should map to Switch-Process' {
+        $alias = Get-Command exec
+        $alias | Should -BeOfType [System.Management.Automation.AliasInfo]
+        $alias.Definition | Should -BeExactly 'Switch-Process'
+    }
+
+    It 'Exec by itself does nothing' {
+        exec | Should -BeNullOrEmpty
+    }
+
+    It 'Exec given a cmdlet should fail' {
+        { exec Get-Command } | Should -Throw -ErrorId 'CommandNotFound,Microsoft.PowerShell.Commands.SwitchProcessCommand'
+    }
+
+    It 'Exec given an exe should work' {
+        $id, $uname = pwsh -noprofile -noexit -outputformat text -command { $pid; exec uname }
+        { Get-Process -Id $id -ErrorAction Stop } | Should -Throw -ErrorId 'NoProcessFoundForGivenId,Microsoft.PowerShell.Commands.GetProcessCommand'
```

Comment:
We could check for empty without throw.

Reply:
Yeah, might be slightly cleaner.  Will update.